rasa/data.py

```diff
@@ -167,6 +168,27 @@ def is_story_file(file_path: Text) -> bool:
         return False
 
 
+def is_end_to_end_conversation_test_file(file_path: Text) -> bool:
+    """Checks if a file is an end-to-end conversation test file.
+
+    Args:
+        file_path: Path of the file which should be checked.
+
+    Returns:
+        `True` if it's a conversation test file, otherwise `False`.
+    """
+
+    if not file_path.endswith(".md"):
```

Comment:
optional: declare a constant for `".md"` at the top of the module.

Reply:
Added into this PR!